.circleci/config.yml

```diff
@@ -139,10 +139,10 @@ jobs:
           command: rustc --version; cargo --version; rustup --version
       - run:
           name: Install mdbook
-          command: |
-              wget https://github.com/rust-lang-nursery/mdBook/releases/download/v0.2.1/mdbook-v0.2.1-x86_64-unknown-linux-gnu.tar.gz
-              tar -xvf mdbook-v0.2.1-x86_64-unknown-linux-gnu.tar.gz
-              mv mdbook /usr/local/cargo/bin/mdbook
+          # The latest release of mdbook has a bug that produces broken
```

Comment:
Is this bug reported somewhere? I'd like to followup on that.

Reply:
I don't now specifically which PR resolved it, but it's already resolved.  The most recent tag is 0.2.3, but the most recent release on the Github releases page is 0.2.1, so maybe autodeployment (if they have it) is broken?

Reply:
[Filed a bug](https://github.com/rust-lang-nursery/mdBook/issues/949)